index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="basic.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <!--- toolbar section -->
+    <nav class="toolbar"> 
+        <div class="fontFam textcolor">Tahseen Zahra</div>
+        <button class="dropdown">
+            <img src="./images/Dropdown.svg" alt="drop down icon">
+        </button>
+        <div class="hov">
+            Zahra's Portfolio
+        </div>
+    </nav>
```

Comment:
- [ ] Kindly place this in a `header` tag as this serves as your navigation bar. 
- [ ] Also, it would be nice you move the Logo holding your name out of the `nav` tag as the `nav` tag is meant to hold the hamburger menu, and in the future your nav links. 
- [ ] Please place the Logo in an `a` tag instead of the `div`  tag you are using. It's a common convention and good practice to make your logo always be a link to the app's home page. And to achieve that, you should place the logo whether it's a text or image, or icon in an `a` tag.

Reply:
@AmaduKamara thanks for reviewing my repo. I am having a bit confusion do I have to enclose all icons like dropdown, GitHub, LinkedIn, and others